scripts/clean.bash

```
@@ -0,0 +1,11 @@
+# Clean workspace
+flutter clean
+
+# Clean packages
+for dir in packages/*; do
+  if [ -f "$dir/pubspec.yaml" ]; then
+    pushd $dir
```

Shellcheck (0.10.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)